xsuite/src/cli/cmd.test.ts

```diff
@@ -94,6 +98,49 @@ test("new-wallet --wallet wallet.json --password 1234", async () => {
   ]);
 });
 
+test("new-wallet --wallet wallet.json --password 1234 --from-pem wallet.pem", async () => {
+  const walletPath = path.resolve("wallet.json");
+  await run(
+    `new-wallet --wallet ${walletPath} --password 1234 --from-pem ${pemPath}`,
+  );
+  const keystore = Keystore.fromFile_unsafe(walletPath, "1234");
+  const secretKey = UserSecretKey.fromPem(fs.readFileSync(pemPath, "utf8"));
+  expect(keystore.getSecretKey()).toEqual(secretKey.hex());
+});
+
+test("new-wallet --wallet wallet.json --password 1234 --from-wallet keystore_key.json", async () => {
+  const walletPath = path.resolve("wallet.json");
+  input.inject("qpGjv7ZJ9gcPXWSN");
+  await run(
+    `new-wallet --wallet ${walletPath} --password 1234 --from-wallet ${keyKeystorePath}`,
+  );
+  const newKeystore = Keystore.fromFile_unsafe(walletPath, "1234");
+  const oldKeystore = Keystore.fromFile_unsafe(
+    keyKeystorePath,
+    "qpGjv7ZJ9gcPXWSN",
+  );
+  const [newSignature, oldSignature] = await Promise.all([
+    newKeystore.newSigner().sign(Buffer.from("Arda ftw")),
```

Comment:
Very kind to put this! But let's have a more neutral text such as "hello" haha

Same for the second test